### DISCUSSION TITLE
Assignees in Beta Project Tasks?

### DISCUSSION THREAD

**Discussion by KrisNey-MSFT** [community/community#13387] [Projects and Issues]
Hello, I am unable to associate an Assignee to a (Beta) Draft Project Task.  This Project task list is comprised of ideas we need to work on, not yet linked to an Issue or PR.  Is this possible? 
Thank you, Kristina

**Comment by Eug3nicekingiceking** [comment]
No

**Reply by KrisNey-MSFT** [reply]
A succinct answer, thank you.  So it looks like we will need to create / convert to actual Issues.  This wasn't exactly the way we wanted to create and track...but we can try (if you have tips, LMK :) ).  Thank you very much, KrisNey-MSFT
